src/libraries/System.Console/src/System/IO/Net6KeyParser.cs

```diff
@@ -0,0 +1,166 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO;
+
+internal static class Net6KeyParser
```

Comment:
Is this class used anywhere at the moment? I only see a commented out reference in `KeyParserTests.cs`.

Reply:
I've added the .NET 6 compat switch (possibility to use the old parsing logic on demand) and it's now used.